packages/assert/src/assert.js

```diff
@@ -52,24 +53,24 @@ const declassifiers = new WeakSet();
  * @param {*} payload What to declassify
  * @returns {StringablePayload} The declassified payload
  */
-function openDetail(payload) {
+function q(payload) {
   const result = harden({
     payload,
     toString() {
-      return payload.toString();
+      return JSON.stringify(payload);
```

Comment:
I believe this will throw RangeError for payloads that contain cyclic graphs.

Reply:
You meant a JSON replacer, right?

Reply:
Indeed!